plugins/python/src/main/kotlin/io/paddle/plugin/python/dependencies/interpretator/AbstractInterpreterDownloader.kt

```diff
@@ -0,0 +1,95 @@
+package io.paddle.plugin.python.dependencies.interpretator
+
+import io.paddle.plugin.python.extensions.pyLocations
+import io.paddle.plugin.python.utils.*
+import io.paddle.project.PaddleProject
+import io.paddle.tasks.Task
+import kotlinx.coroutines.runBlocking
+import org.codehaus.plexus.util.Os
+import java.io.File
+
+internal sealed class AbstractInterpreterDownloader(private val userDefinedVersion: InterpreterVersion, private val project: PaddleProject) {
```

Comment:
Why sealed, not abstract (just curious)? What if smb else wants to implement his own interpreter in plugin for paddle later?